javav2/example_code/rds/Readme.md

```diff
@@ -35,6 +63,13 @@ Define these values to successfully run the JUnit tests:
 - **masterUsername** - The master user name .
 - **masterUserPassword** - The password that corresponds to the master user name.
 - **newMasterUserPassword** - The updated password that corresponds to the master user name.
```

Comment:
Recommend we avoid using the term "master".

Reply:
Took out this word from the description